.gitattributes

```diff
@@ -1 +1,2 @@
-* -crlf
\ No newline at end of file
+* -crlf
+src/psake.ps1 text
```

Comment:
Maybe this project should use a more typical .gitattributes e.g.:
```
CHANGELOG.md merge=union
* text=auto
```

Reply:
Reported a new issue: #266